examples/medplum-patient-intake-demo/data/core/patient-intake-questionnaire.json

```diff
@@ -47,6 +47,62 @@
         }
       }
     },
+    {
+      "fullUrl": "urn:uuid:183a569a-7f6d-41f9-97d8-57691fbc2876",
+      "request": {
+        "method": "PUT",
+        "url": "ValueSet?name=smoking-status-codes"
+      },
+      "resource": {
+        "resourceType": "ValueSet",
+        "status": "active",
+        "url": "http://example.com/smoking-status",
```

Comment:
I took a look at the [US Core Smoking Status Observation Profile](http://hl7.org/fhir/us/core/StructureDefinition/us-core-smokingstatus) and tried to use the `SmokingStatusComprehensive` ValueSet but I got a not found error. So I used the same options from the [SmokingStatus component](https://github.com/medplum/medplum/blob/3844220953b3ccde399dc52d5cb8390d361571d9/packages/react/src/PatientSummary/SmokingStatus.tsx#L14-L23).

![image](https://github.com/user-attachments/assets/0e20e93a-1e23-418a-8396-c45fbe0293e7)

Reply:
@pamella  you might be looking at a later version of US Core. Right now, we implement [U.S. Core 5.0.0,](https://hl7.org/fhir/us/core/STU5/StructureDefinition-us-core-smokingstatus.html) which uses _this_ :

http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113883.11.20.9.38

Reply:
See my comment about the US State valuset (using `expansion`, version `0.0.0`) to create this valueset as well